ui/tests/acceptance/clients/counts/sync-test.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import clientsHandler from 'vault/mirage/handlers/clients';
+import { visit, click, currentURL } from '@ember/test-helpers';
+import authPage from 'vault/tests/pages/auth';
+import { SELECTORS } from 'vault/tests/helpers/clients';
+
+module('Acceptance | clients | sync', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function () {
+    clientsHandler(this.server);
+    await authPage.login();
+    return visit('/vault/clients/counts/sync');
+  });
+
+  test('it should render charts when secrets sync is activated', async function (assert) {
+    assert.expect(3);
+    this.server.get('/sys/activation-flags', () => {
+      assert.true(true, '/sys/activation-flags/ is called to check if secrets-sync is activated');
+
+      return {
+        data: {
+          activated: ['secrets-sync'],
+          unactivated: [],
+        },
+      };
+    });
+
+    assert.dom(SELECTORS.charts.chart('Secrets sync usage')).exists('Secrets sync usage chart is rendered');
+    assert.dom(SELECTORS.syncTab.total).exists('Total sync clients chart is rendered');
```

Comment:
TODO need to double check if these are the appropriate assertions to make; these are just guesses

Reply:
In case it's helpful, this PR has screenshots of how the sync tab looks with mock data https://github.com/hashicorp/vault/pull/25669